presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorPageSourceProvider.java

```diff
@@ -69,7 +69,7 @@ public ConnectorPageSource createPageSource(
         Optional<Map<String, Type>> columnTypes = raptorSplit.getColumnTypes();
         boolean tableSupportsDeltaDelete = raptorSplit.isTableSupportsDeltaDelete();
 
-        FileSystemContext context = new FileSystemContext(session);
+        HdfsContext context = new HdfsContext(session.getIdentity());
```

Comment:
So does here.

Reply:
We can ignore schema and table for page source/sink provider at the moment

Reply:
Same